zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -5156,37 +5357,42 @@ components:
           $ref: "#/components/schemas/SortOrderEnum"
       required:
         - field
-    FlowNodeInstanceSearchQueryRequest:
-      description: Flow node instance search request
+    FlowNodeInstanceSearchQueryRequestBase:
+      description: Base properties for FlowNodeInstanceSearchQueryRequest
+      type: object
       allOf:
         - $ref: "#/components/schemas/SearchQueryRequest"
-      type: object
       properties:
         sort:
           description: Sort field criteria.
           type: array
           items:
             $ref: "#/components/schemas/FlowNodeInstanceSearchQuerySortRequest"
+    FlowNodeInstanceSearchQueryRequest:
+      description: Flow node instance search request
+      deprecated: true
+      type: object
+      allOf:
+        - $ref: "#/components/schemas/FlowNodeInstanceSearchQueryRequestBase"
+      properties:
         filter:
           description: The flow node instance search filters.
           allOf:
             - $ref: "#/components/schemas/FlowNodeInstanceFilterRequest"
-    FlowNodeInstanceFilterRequest:
-      description: Flow node instance filter.
+    FlowNodeInstanceSearchQuery:
+      description: Flow node instance search request Key attributes as string values.
```

```suggestion
      description: Flow node instance search request key attributes as string values.
```

Nitpick -- non-blocking.

---

Nice catch, that should be actually a separate sentence! 😀
I'll fix that.